### PR TITLE
GAWB-2928 test tags and consent code columns display correctly

### PR DESCRIPTION
**Pull request by ahaessly**
test correct tags and consent codes display in the table on the data library page

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala
Comment (aednichols) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala]: Intentional?

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala
Comment (aednichols) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala]: This `Any` could become an `Either[String, Seq[String]]`

Unfortunately, the changes are far reaching and add a lot of syntax:

```
val tags: Map[String, Either[String, Seq[String]]] = Map(
  "tag:tags" -> Right(Seq("testing","diabetes"))
)
```
and
```
val testAttributes = Map("A-key" -> Left("A value"), "B-key" -> Left("B value"), "C-key" -> Left("C value"))
```
So feel free to ignore this suggestion.

**Review by aednichols — COMMENTED** [review]
Code seems to be doing the right thing, failing for me locally though. Will wait for Jenkins.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala
Comment (ahaessly) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala]: nope. thanks!

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala
Comment (ahaessly) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala]: i like the idea, but i'm using a pretty simple object to json mapper to send the data and my tags map gets serialized to this: {"tag:tags":{"r":["testing","diabetes"]}}

**Comment by aednichols** [comment]
Build above was aborted, possibly because of quotas, which should have reset

jenkins retest

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala
Comment (aednichols) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala]: Ah, hadn't come across that

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala
Comment (aednichols) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala]: Here's another print that I missed the first time around

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala
Comment (davidangb) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala]: we'd really want this to be `Map[String, Attribute]` I think ... but that would require pulling in rawls-model, which is bigger than desired for this PR. How about just add a comment explaining what you expect the `Any` to be?

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Tags.scala
Comment (davidangb) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Tags.scala]: hunting for divs feels a little fragile, in case we ever change the layout. Alternative would be to update `render-tags`/`render-tag` to include a class or data-test-id or the like. I'm fine either way for this PR, your call.

**Review by davidangb — APPROVED** [review]
some comments inline but nothing to hold the PR. :+1:

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Tags.scala
Comment (aednichols) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Tags.scala]: There is code to do exactly that in my PR. Relevant part
```
([tag] (render-tag {:data-test-id (str (string/lower-case tag) "-tag")} tag))
```

**Review by aednichols — APPROVED** [review]
👍 LGTM, OK to merge when extraneous print is removed

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Tags.scala
Comment (ahaessly) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Tags.scala]: chose a different solution - using the title

**Review by aednichols — APPROVED** [review]
Still 👍

**Review by davidangb — APPROVED** [review]
:+1: lgtm, just need to see tests pass!

**Pull request merged**
